tests/min-reporter.cjs

```diff
@@ -0,0 +1,10 @@
+const { reporters } = require('mocha');
```

Comment:
Unfortunately this file has to be cjs, mocha does not support esm reporter files.